src/style.css

```
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center left;
+  box-sizing: border-box;
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+}
+
+.stars--5 .stars__star {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
```

it's better to combine them together

```suggestion
.stars--5 .stars__star,
.stars--1 .stars__star:nth-child(1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4) {
  background-image: url("images/star-active.svg");
}
```